Examples/MAX32520/Coremark/README.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+This example determines the Coremark rating for the MAX32xxx and MAX78xxx series microcontrollers.
+
```

Comment:
Please add the following contents to the README, which have now become a part of the new [standard readme template in MSDKGen](https://github.com/Analog-Devices-MSDK/MSDKGen/tree/main/Markdown/Template).

```Markdown
### Project Usage

Universal instructions on building, flashing, and debugging this project can be found in the **[MSDK User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/)**.

### Project-Specific Build Notes

(None - this project builds as a standard example)
```

This project doesn't offer any build config options, but you may want to expand a bit on exactly where the source code is pulled from in the build notes section.